tests/apps/users/factories.py

```diff
@@ -0,0 +1,32 @@
+from jobbergate_api.apps.users.models import User
+
+
+class BaseFactory:
```

Comment:
Not sure I understand the purpose of this. this *looks* like a dataclass, and if so you could just use `pydantic.BaseModel` for that, but 
- you could also just use a unittest.mock.Mock object for that, but also
- why not just use the actual User object you already have in the app?